openslides/utils/collection.py

```diff
@@ -102,8 +129,16 @@ def get_instance(self):
         """
         if self.is_deleted():
             raise RuntimeError("The collection element is deleted.")
+
         if self.instance is None:
-            self.instance = self.get_model().objects.get(pk=self.id)
+            # The config instance has to be get from the config element, because
+            # some config values are not in the db.
+            from openslides.core.config import config
+            if (self.collection_string == config.get_collection_string() and
+                    isinstance(self.id, str)):
```

Comment:
When a config value is changed, the save method a ConfigStore-element is called and therefore the autoupdate system is called with this ConfigStore-element. In this case the id is the numeric id of this db-element.